java/sqlcommenter-java/README.md

```diff
@@ -110,12 +110,35 @@ or alternatively just for a specific method
 
 #### Hibernate
 
+##### Hibernate via JPA
+
 If you're using Hibernate via JPA, then you can simply set the `hibernate.session_factory.statement_inspector` configuration property in the `persistence.xml` configuration file:
 
 ```xml
 <property name="hibernate.session_factory.statement_inspector" value="com.google.cloud.sqlcommenter.schibernate.SCHibernate" />
 ```
 
+##### Hibernate via Spring Application Properties
+If you're using Hibernate via Spring application properties(i.e. you have `application.yml` or `application.properties` in your resources folder)
+- `application.properties`
+
+```properties
+spring.jpa.properties.hibernate.session_factory=statement_inspector: com.google.cloud.sqlcommenter.schibernate.SCHibernate
```

Comment:
I am not much aware of spring properties, but is this format right or should it be:
```
spring.jpa.properties.hibernate.session_factory.statement_inspector=com.google.cloud.sqlcommenter.schibernate.SCHibernate
```
?

Reply:
Yeah my bad, I put it wrongly.